src/PiWeb.Import.Sdk/Modules/ImportAutomation/IImportRunner.cs

```diff
@@ -17,7 +17,7 @@ namespace Zeiss.PiWeb.Import.Sdk.Modules.ImportAutomation;
 /// <summary>
 /// Responsible for running custom import logic.
 /// </summary>
-public interface IImportRunner : IAsyncDisposable
+public interface IImportRunner : IDisposable

```

Comment:
Should we actually change this to `IDisposable`?
Since there is a `RunAsync` I could imagine the developer might want to wait for an asynchronous operation in the dispose method. 
Or do we wait for `RunAsync` to return before we call `Dispose`?

Reply:
I changed it to synchronous dispose because the factory method creating the runner is also not async mostly because I think we have a much easier time implementing the SDK when not everything is async. Since you can do your async initialization and cleanup at the beginning and the end of the run method, I think this is fine. Also the dispose is always called after the run method completed.

Reply:
Then I'm fine with that change.